src/test/java/domain/piece/strategy/InnerElephantStrategyTest.java

```diff
@@ -0,0 +1,29 @@
+package domain.piece.strategy;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import domain.TeamType;
+import domain.piece.Piece;
+import domain.piece.PieceType;
+import domain.position.Position;
+import java.util.Map;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class InnerElephantStrategyTest {
+
+    @Test
+    @DisplayName("Inner Elephant 형식으로 마와상을 반환한다")
+    void testCreateElephantHorse() {
+        // given
+        InnerElephantStrategy innerElephantStrategy = new InnerElephantStrategy();
+
+        // when
+        Map<Position, Piece> choElephant = innerElephantStrategy.createElephantHorse(TeamType.CHO);
+
+        // then
+        Position elephantPosition = Position.of(0, 2);
+        assertThat(choElephant.get(elephantPosition).isSameType(PieceType.ELEPHANT));
```

Comment:
`assertThat` 이후 아무 검사를 하고 있지 않습니다. 
아래처럼 `isTrue()`까지 호출해야 의도된 검사가 됩니다.
다른 `StrategyTest` 들도 모두 점검해주세요 :)

```suggestion
        assertThat(choElephant.get(elephantPosition).isSameType(PieceType.ELEPHANT)).isTrue();
```